app/src/screens/Developer.tsx

```diff
@@ -251,6 +252,14 @@ const Settings: React.FC = () => {
     setEnableShareableLink((previousState) => !previousState)
   }
 
+  const toggleEnableProxySwitch = () => {
+    dispatch({
+      type: BCDispatchAction.TOGGLE_PROXY,
+      payload: [!enableProxy],
+    })
+    setEnableProxy((previousState) => !previousState)
```

Comment:
do we need to re-initialize the agent here or something so that the proxy gets activated/deactivated when toggled? or are we okay with restarting the app?

Reply:
I think for now we're okay with restarting the app